kernel/banner.c

```diff
@@ -30,6 +30,10 @@ void boot_banner(void)
 #endif /* defined(CONFIG_BOOT_DELAY) && (CONFIG_BOOT_DELAY > 0) */
 
 #if CONFIG_BOOT_BANNER
-	printk("*** Booting Zephyr OS build " BANNER_VERSION BANNER_POSTFIX " ***\n");
-#endif /* CONFIG_BOOT_BANNER */
+	if (strlen(CONFIG_BOOT_BANNER_OVERRIDE) > 0) {
+		printk("*** Booting Zephyr OS build " CONFIG_BOOT_BANNER_OVERRIDE " ***\n");
```

Comment:
Maybe I did not explain it well - what I mean is with this, why should this display `Booting Zephyr OS build` at all? That text is what `CONFIG_BOOT_BANNER_OVERRIDE` should replace, for example for a downstream use e.g. ncs it could display `Booting NCS OS build`. There is currently a big problem with people posting bugs to zephyr that have nothing to do with zephyr and are NCS bugs, this is a great way to help alleviate that problem and I like the idea.

Reply:
Ah, sure, it was an oversight.